src/status_im/contexts/wallet/account/bridge/view.cljs

```diff
@@ -1,42 +1,34 @@
 (ns status-im.contexts.wallet.account.bridge.view
   (:require
     [quo.core :as quo]
-    [quo.foundations.resources :as quo.resources]
     [react-native.core :as rn]
+    [reagent.core :as reagent]
     [status-im.contexts.wallet.account.bridge.style :as style]
     [status-im.contexts.wallet.common.account-switcher.view :as account-switcher]
-    [status-im.contexts.wallet.common.temp :as temp]
+    [status-im.contexts.wallet.common.asset-list.view :as asset-list]
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
-(defn network-logo
-  [item]
-  {:source (quo.resources/get-network (:network-name item))})
-
-(defn- bridge-token-component
-  []
-  (fn [token]
-    (let [on-press #(rf/dispatch [:wallet/select-bridge-to
-                                  {:token    token
-                                   :stack-id :wallet-bridge}])]
-      [quo/token-network (assoc token :on-press on-press)])))
-
 (defn view
   []
-  (let [networks       (rf/sub [:wallet/network-details])
-        networks-logos (map network-logo networks)]
-    [rn/view {:style {:flex 1}}
-     [account-switcher/view
-      {:on-press            #(rf/dispatch [:navigate-back])
-       :accessibility-label :top-bar}]
-     [quo/text-combinations
-      {:container-style style/header-container
-       :title           (i18n/label :t/bridge)}]
-     [quo/input
-      {:container-style style/input-container
-       :icon-name       :i/search
-       :placeholder     (i18n/label :t/search-assets)}]
-     [rn/flat-list
-      {:data                    (temp/bridge-token-list networks-logos)
-       :render-fn               bridge-token-component
-       :content-container-style style/list-content-container}]]))
+  (let [search-text (reagent/atom "")]
+    (fn []
+      [rn/view {:style {:flex 1}}
+       [account-switcher/view
+        {:on-press            #(rf/dispatch [:navigate-back])
```

Comment:
is this the right action @briansztamfater ?

Reply:
As it is the first screen in the flow `:navigate-back` should work because it dismisses the current modal if there's any.

https://github.com/status-im/status-mobile/blob/602b27105bcc4960a8748dd60af6bdc8967559c0/src/status_im/navigation/effects.cljs#L91-L95